### ISSUE TITLE
Rats Mod ALL Rats are TAMED by default... can not tame. 

### ISSUE THREAD

**Issue by chasethemillion** [adam9899/Issues#117]
My friend and I are playing on a private server and neither of us have used Rats before. We are starting to give it a try... but we cant seem to find any rats that do not already say "Tamed" in their info box hwaila. 

When we give them cheese it says that we can not tame these rats.... any ideas why?

**Comment by ByteTemplar** [comment]
Yes, its known bug that Pipers rats stay in the world even though it despawns himself, leaving huge amounts of tamed rats in the world. Especially in chunkloaded areas, like players bases unfortunately. Even in single player this is a problem, I had almost 1000 rats under my base at one point!

You can only kill em by using fire (lava bucket, flint&steel, Ignite spell etc...).
You can kill em by command, but be careful with it and not kill your own!
Garbage block can be used to spawn normal, tameable rats btw, just a tip...

https://github.com/Alex-the-666/Rats/issues/150

**Comment by wezu123** [comment]
I'm having the same issue, every rat is tamed, and immortal.

**Comment by ByteTemplar** [comment]
> immortal.

Like I said in previous message, you can kill em by fire if needed...
